tests/testthat/test-load-csv.R

```diff
@@ -14,3 +14,12 @@ test_that("SEA CSV loads in", {
   expect_gt(nrow(load_sea_file(sample_sea)), 0)
   expect_false(any(endsWith(load_sea_file(sample_sea)[[COL_MODULE]], ".")))
 })
+
+test_that("load_ace_file returns raises warning when called on a file with all practice data - should return an empty dataframe", {
+  all_practice_file_path = file.path(aceR_sample_data_path("nexus"), "all_practice_backwardsspatialspan.csv")
+  warnings <- capture_warnings({
+    result <- load_ace_file(all_practice_file_path, app_type = "nexus")
+  })
+  expect_true(any(grepl("processing resulted in an empty data frame", warnings)))
```

Comment:
Thanks for writing this test as well! Per the current [testthat docs](https://testthat.r-lib.org/reference/expect_error.html), it looks like `capture_warnings()` is superseded, and they recommend this syntax for testing the warning and the returned object:

```suggestion
  expect_warning(
    result <- load_ace_file(all_practice_file_path, app_type = "nexus"),
    "processing resulted in an empty data frame"
  )
```

Then current line 24 that checks to make sure the result has 0 rows should still run as usual, because `result` will be assigned in the process of running `expect_warning()`.